### PR TITLE
#34  Improve contribution.md file. Solved!

### PR DESCRIPTION
**Pull request by neokd**
Updated the contribution.md file with new folder structure.

### CODE REVIEW

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
image

In the same file, I have added some snippets and defined the path as well. Do change that as well. Rest looks good.

**Comment by neokd** [comment]
Do i need to add the entire snippet ?

File: contribution.md [hunk 1]
```diff
@@ -65,7 +62,7 @@ Now you are done with the project setup, now you can make the changes you want o
 ### Let say you want to scrape the avatar url of and user. Steps applying which we can do this
 + At first we have to scrape the profile page of an user. For that we have defined a function in the user class as
 ```python
-# scrape-up/src/scrape_up/github/__init__.py/
+# scrape-up/src/scrape_up/github/respository.py
```
Comment (nikhil25803) [inline contribution.md:65]: Not `repository.py`, but `users.py`. Change it.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by neokd** [comment]
Updated!

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**